src/main/java/org/opentripplanner/model/plan/Itinerary.java

```diff
@@ -663,4 +676,12 @@ public void setEmissionsPerPerson(Emissions emissionsPerPerson) {
   public Emissions getEmissionsPerPerson() {
     return this.emissionsPerPerson;
   }
+
+  private static int penaltyCost(TimeAndCost penalty) {
+    if (penalty == null) {
+      return 0;
+    } else {
+      return penalty.cost().toSeconds();
+    }
```

Comment:
```suggestion
    return (penalty == null ? Cost.ZERO : penalty.cost()).toSeconds();
    }
```
Avoid magic numbers: 0

Reply:
A better way to do this is to make the penalty none null, and initialize it to `TimeAndCost.ZERO`. I do not think we need to differensiate between ZERO and NOT_SET.